### ISSUE TITLE
SSH and username

### ISSUE THREAD

**Issue by Ganiest** [felixse/FluentTerminal#840]
Hello, i'm using your terminal for PS, cmd and SSH.

'Use windows username when opening SSH URLs that don't specify one' is turned off.
I didn't find a place where i could specify a default username for ssh connections.
Setting up a ssh profile with the wished username and a wildcard in SSH profiles didn't do the trick.

Did i miss a setting somewhere or is it a feature you could add?

Best regards.
GL

**Comment by skipmeister123** [comment]
That's something that would go in your ssh config.  Typically, it'd look like this:
home\.ssh\config:
```
Host *
User myname
```

**Comment by Ganiest** [comment]
Created my config file in C:\Users\%USERNAME%\.ssh as you said:

'
Host *
User admin
'

But the terminal is still opening a popup with my username instead of 'admin'.

**Comment by skipmeister123** [comment]
Verify that the file is actually config, and not config.txt (in case your extensions aren't shown)

**Comment by Ganiest** [comment]
The file's name is now without extension.

The credentials popup is still displaying my username instead of admin.

Any idea?

**Comment by skipmeister123** [comment]
Try to connect manually with the verbosity pumped up a bit.  `ssh -vvv host`.  What's the output look like?

**Comment by Ganiest** [comment]
When Terminal is already opened if i open a new tab and call 'ssh iptoconnect', the user is admin and it's ok.
the -vvv ouput is just here 
[new 2.txt](https://github.com/felixse/FluentTerminal/files/5130469/new.2.txt)


But if I go by Start menu > Execute and type ssh://iptoconnect, the credentials popup still displays my username.
![Capture](https://user-images.githubusercontent.com/70203433/91316671-62c0f400-e7b9-11ea-944e-b61af7d68faf.JPG)

**Comment by skipmeister123** [comment]
Yes, the config file looks like it's working.  I think I'd want some more information from @Felix in order to continue troubleshooting.
It's possible that you may need to restart the tray process, but other than that, I don't know what the issue could be.
